### PR TITLE
Index management TOC and naming change

### PR DESCRIPTION
**Pull request by ariamarble**
### Description

Updates naming and TOC structure for Index management in Dashboards pages.

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/im-dashboards/datastream.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
 layout: default
 title: Data stream management
```
Comment (kolchfa-aws) [inline _dashboards/im-dashboards/datastream.md:3]: I would remove the word "management" from the child pages. I would name the child pages "Indexes", "Data streams", "Force merge" and "Rollover"

File: _tuning-your-cluster/availability-and-recovery/snapshots/sm-dashboards.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
 ---
 layout: default
-title: Snapshot management
-parent: Index and snapshot management in OpenSearch Dashboards
-nav_order: 17
+title: Snapshot management in Dashboards
```
Comment (kolchfa-aws) [inline _tuning-your-cluster/availability-and-recovery/snapshots/sm-dashboards.md:3]: I vote for moving this back to Dashboards and linking from here.

Comment (ariamarble) [reply _tuning-your-cluster/availability-and-recovery/snapshots/sm-dashboards.md:3]: I do too. Imma change it

File: _dashboards/sm-dashboards.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-parent: Index and snapshot management in OpenSearch Dashboards
-nav_order: 17
+title: Snapshot management in Dashboards
+nav_order: 90
+redirect_from:
+  - /dashboards/admin-ui-index/sm-dashboards/
 ---
 
-# Snapshot management
+# Snapshot management in Dashboards
```
Comment (kolchfa-aws) [inline _dashboards/sm-dashboards.md:9]: Now that we have everything under Dashboards, I think we can remove "in Dashboards" from Index management and Snapshot management

**Review by kolchfa-aws — APPROVED** [review]
LGTM with comment.

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**